QuizApp/Search/SearchViewController.swift

```diff
@@ -56,6 +57,30 @@ class SearchViewController: UIViewController {
         }
     }
 
+    func textFieldDidBeginEditing(_ textField: UITextField) {
+        textField.layer.borderWidth = 1
+    }
+
+    func textFieldDidEndEditing(_ textField: UITextField) {
+        textField.layer.borderWidth = 0
+    }
+
+    func bindViews() {
+        searchBar.textField
+            .textDidBeginEditing
+            .sink { [weak self] _ in
+                self?.searchBar.textField.layer.borderWidth = 1
+            }
+            .store(in: &cancellables)
+
+        searchBar.textField
+            .textDidEndEditing
+            .sink { [weak self] _ in
+                self?.searchBar.textField.layer.borderWidth = 0
+            }
+            .store(in: &cancellables)
+    }
```

Comment:
do you still need the `func textFieldDidBeginEditing(_ textField: UITextField)` and `func textFieldDidEndEditing(_ textField: UITextField)` now that you have the observers?